### ISSUE TITLE
New memory leak

### ISSUE THREAD

**Issue by claudioandre** [magnumripper/JohnTheRipper#2612]
* detected by mem_dbg on a `--test` run;
* on clang and gcc.

```
Testing: money, Microsoft Money (2002 to Money Plus) [MD5/SHA1 32/64]... (2xOMP) PASS
Mem leak: 512 bytes, alloc_num 19860, file money_fmt_plug.c, line 96
```

**Comment by magnumripper** [comment]
I found and fixed this while working with #2607, will be committed soon.

**Comment by magnumripper** [comment]
Fix in 577b3034f
